tests/test_subprocess.py

```diff
@@ -8,8 +8,8 @@ def test_command():
     sleep05 = Command("sleep 0.5")
 
     sleep05.run(timeout=1)
-    print(sleep05)
-    assert sleep05.retcode == 0
```

Comment:
This unit test seems to fail randomly (appears in Windows alone), I guess that's probably the reason the `print` exists.

First time failure: https://github.com/materialsvirtuallab/monty/actions/runs/10104741275/job/27944181978

Second time: https://github.com/materialsvirtuallab/monty/actions/runs/10104606892/job/27943837091